openstates/ri/events.py

```diff
@@ -59,14 +59,20 @@ def scrape_agenda(self, url, session):
             "%A, %B %d, %Y %I:%M",
         ]
 
+        kwargs = {}
         if time in all_day:
             datetime = date
         else:
             datetime = "%s %s" % ( date, time )
-        if "CANCELLED" in datetime or "Rise of the House" in datetime:
-            # XXX: Do something more advanced.
+        if "CANCELLED" in datetime:
             return
 
+        if "Rise of the" in datetime:
+            datetime = date
+            kwargs["all_day"] = True
+            kwargs["time_text"] = time
```

Comment:
just pushed a change, it's now in the description, which should read something like "Meeting Notice: Starting at Rise of the Joint Health & Human Services/HEW Hearing" for these weird ones.